privacyidea/static/components/config/views/config.realms.html

```diff
@@ -9,6 +9,12 @@
                 <li><a ng-click="clearDefaultRealm()"
                        ng-show="checkRight('resolverwrite')"
                         translate>Clear default realm</a></li>
+                <hr>
+                <li role="presentation">
+                        <a target="_external" ng-href="{{ 'https://privacyidea.readthedocs.io/en/v'+
+                           privacyideaVersionNumber+'/configuration/realms.html#'}}">
```

Comment:
same here

Reply:
done in b5be0d720e77db9fe1eab889e9384eb1e4c7a521.